packages/encryption/src/lib/schemas/index.ts

```diff
@@ -0,0 +1,28 @@
+import { JSONSchemaType } from 'ajv';
+
+import { LIT_ERROR } from '@lit-protocol/constants';
+import { throwError } from '@lit-protocol/misc';
+import { ConditionType } from '@lit-protocol/types';
+
+export const SCHEMAS: { [K in ConditionType]: JSONSchemaType<any> } = {
+  cosmos: require('./LPACC_ATOM.json'),
+  evmBasic: require('./LPACC_EVM_BASIC.json'),
+  evmContract: require('./LPACC_EVM_CONTRACT.json'),
+  solRpc: require('./LPACC_SOL.json'),
```

Comment:
nit: can we load these async? using require seems a bit dangerous. Perhaps we use `import` and `Promise.all` to concurrently load schemas instead.

Reply:
Done on the other repo